.vscode/settings.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
please remove this directory. to avoid committing it in the future you could consider adding it to `.gitignore`

Reply:
Done

Reply:
you have to actually remove this directory from git though. `git rm -rf .vscode` and then commit that. this will also delete the contents of the directory so you should back it up first if you dont want to regenerate it with your IDE